components/about/experiences/display.module.tsx

```diff
@@ -0,0 +1,200 @@
+import styles from "./display.module.scss";
+import { motion } from "framer-motion";
+import timeSince from "@components/functions/time_since.module";
+import { TagExperience } from "@typeFiles/api/tag.type";
+
+
+const MEDIA = process.env.NEXT_PUBLIC_MEDIA_SERVER;
+
+export default function ExperienceDisplayComponent(props: Props): JSX.Element {
+  const { current, tagCount, changeIndex } = props;
+  const lastUpdated = Number(new Date(current.icon_updated_at));
+
+  const skillConfidence = (score: number): string => {
+    if (score <= 10) {
+      return "Interested";
+    } else if (score <= 20) {
+      return "Beginner";
+    } else if (score <= 30) {
+      return "Novice";
+    } else if (score <= 50) {
+      return "Intermediate";
+    } else if (score <= 90) {
+      return "Experienced";
+    } else {
+      return "Advanced";
+    }
+  };
+
+  const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+  const date = new Date(current?.used_since);
+
+  const animation_list = {
+    hidden: { opacity: 0, x: 100 },
+    visible: { opacity: 1, x: 0 },
+  };
+
+  const animation_icon = {
+    hidden: { opacity: 0, rotate: 70 },
+    visible: { opacity: 1, rotate: 0 },
+  };
+
+  const img =
+    <motion.img
+      src={`${MEDIA}/tag/${current.id}.svg?last_updated=${lastUpdated}`}
+      key={current.id}
+      width={64}
+      height={64}
+      alt={current.name}
+
+      // Animation
+      initial="hidden"
+      animate="visible"
+      variants={animation_icon}
+      transition={{ type: "spring", stiffness: 260, damping: 15 }}
+    />;
+
+  return (
+    <article
+      className={styles.wrapper}
+    >
+
+      <section className={styles.header}>
+        {tagCount > 1 && <button className={styles.button} onClick={() => changeIndex(-1)}>&lt;</button>}
+
+        {img}
```

Comment:
Nitpick. Not really a fan of these variable tsx. Should be a dumb function component.

Reply:
Or just inline.